uber/config.py

```diff
@@ -189,7 +205,7 @@ def _unrepr(d):
 
 c.BADGE_PRICES = _config['badge_prices']
 for _opt, _val in chain(_config.items(), c.BADGE_PRICES.items()):
-    if not isinstance(_val, dict):
+    if not isinstance(_val, dict) and not hasattr(c, _opt.upper()):
         setattr(c, _opt.upper(), _val)
```

Comment:
@EliAndrewC  take note: 

setattr(c, "SQLALCHEMY_URL") was crashing because I had already done ```def @property SQLALCHEMY_URL()``` in the class itself.

this I guess is the first time we've ever overridden a config-file generated property, the default behavior before would have been for it to crash.  since it wasn't crashing here before, I figure that this is OK behavior.  

it does have a side effect though: if we ever define something explicitly in the class it will ALWAYS mask the config data automatically generated attribute.  which..... seems correct to me (and better than crashing).  just, if we ever rename stuff en-masse we need to be careful that we don't accidentally override something we didn't mean to.